.travis.yml

```diff
@@ -5,12 +5,16 @@ d: dmd-2.071.0
 install:
  - wget "https://raw.githubusercontent.com/dlang/dmd/master/src/checkwhitespace.d" -O ../checkwhitespace.d
  - git clone https://github.com/Hackerpilot/Dscanner
- - (cd Dscanner && git checkout tags/v0.4.0-alpha2)
+ - (cd Dscanner && git checkout tags/v0.4.0-alpha4)
  - (cd Dscanner && git submodule update --init --recursive)
- - (cd Dscanner && make githash debug)
+ # debug build is faster, but disable 'missing import' messages (missing core from druntime)
+ - (cd Dscanner && sed 's/dparse_verbose/StdLoggerDisableWarning/' -i makefile && make githash debug)
```

Comment:
imports to `core` won't resolve (cloning this won't solve the problem, as it has imports to `gcc/builtins` and `ldc/intrinsics`)